lib/mempool/mempool.js

```diff
@@ -1326,6 +1321,9 @@ class Mempool extends EventEmitter {
     if (!proof.isSane())
       throw new VerifyError(proof, 'invalid', 'bad-airdrop-proof', 100);
 
+    if (this.nextState.hasAirstop)
+      throw new VerifyError(proof, 'invalid', 'bad-airdrop-disabled', 0);
```

Comment:
I think you can actually enforce this policy rule by default as part of the release. This is a bit different than Bitcoin soft forks that add a feature but I think you can see in Taproot that mempool policy was implemented without any activation:

https://github.com/bitcoin/bitcoin/commit/e9a021d7e6a454d610a45cb9b3995f0d96a5fbb6

This at least rules out any race condition or off-by-one issue with the miners. Anyone who really wants to get a claim in still can - it just be hard to relay as the next release is deployed.

Reply:
After soft-fork activates, I believe this one needs to be in place. It's will no longer be an option to have bad mempool (Can't create a block if there are airdrops in it)

Policy on top could definitely help (Some could preemptively disallow those), but enabling by default would mean that you will have to contact miners to get airdrops in?

I am not 100% sure that's what you meant ?

If I understood btc commit properly, that PR adds new checks for newly introduced thing, does not disallow old thing ?